### PR TITLE
Commerce 8505 High Prices break UI

### PR DESCRIPTION
**Pull request by christian-chiappa**

### CODE REVIEW

**Comment by christian-chiappa** [comment]
ci:test:sf

**Comment by christian-chiappa** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-api/src/main/java/com/liferay/commerce/constants/CommercePriceConstants.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+package com.liferay.commerce.constants;
+
+/**
+ * @author Christian Chiappa
+ */
+public class CommercePriceConstants {
+
+	public static final double COMMERCE_MAX_PRICE_VALUE = 999999999.99;
+
+	public static final int COMMERCE_MIN_PRICE_VALUE = 0;
```
Comment (luca-pellizzon) [inline modules/apps/commerce/commerce-api/src/main/java/com/liferay/commerce/constants/CommercePriceConstants.java:24]: Why are we using different types here? What if we want the minimum price to be 15.50?

File: modules/apps/commerce/commerce-product-definitions-web/src/main/resources/META-INF/resources/instance/details.jsp [hunk 2]
```diff
@@ -32,6 +32,8 @@ if ((cpInstance != null) && (cpInstance.getExpirationDate() != null)) {
 }
 
 boolean discontinued = BeanParamUtil.getBoolean(cpInstance, request, "discontinued");
+int minPriceValue = CommercePriceConstants.COMMERCE_MIN_PRICE_VALUE;
+double maxPriceValue = CommercePriceConstants.COMMERCE_MAX_PRICE_VALUE;
```
Comment (luca-pellizzon) [inline modules/apps/commerce/commerce-product-definitions-web/src/main/resources/META-INF/resources/instance/details.jsp:36]: Don't need to define varables here, use the constants directly where needed.

File: modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/service/impl/CPInstanceLocalServiceImpl.java [hunk 3]
```diff
@@ -1234,6 +1236,16 @@ public CPInstance updatePricingInfo(
 			BigDecimal cost, ServiceContext serviceContext)
 		throws PortalException {
 
+		BigDecimal maxValue = BigDecimal.valueOf(
+			CommercePriceConstants.COMMERCE_MAX_PRICE_VALUE);
+
+		if ((price.compareTo(maxValue) > 0) ||
```
Comment (luca-pellizzon) [inline modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/service/impl/CPInstanceLocalServiceImpl.java:1242]: We are defining a min-price-value in the JSPs, but we are not validating it here.

**Review by luca-pellizzon — CHANGES_REQUESTED** [review]
There are places where we define a min-price but then this is not validated.
It wasn't something in the AC, right?

**Comment by christian-chiappa** [comment]
ci:test:sf

**Review by luca-pellizzon — APPROVED** [review]

**Comment by luca-pellizzon** [comment]
ci:test:sf

**Comment by luca-pellizzon** [comment]
ci:test:relevant

**Comment by christian-chiappa** [comment]
ci:test:relevant

**Comment by 22LucaGiannotta22** [comment]
ci:forward

**Pull request closed without merging**